devlib/module/cpufreq.py

```diff
@@ -313,3 +313,44 @@ def set_max_frequency(self, cpu, frequency, exact=True):
             self.target.write_value(sysfile, value)
         except ValueError:
             raise ValueError('Frequency must be an integer; got: "{}"'.format(frequency))
+
+    def set_governor_for_cpus(self, cpus, governor, **kwargs):
+        """
+        Set the governor for the specified list of CPUs.
+        See https://www.kernel.org/doc/Documentation/cpu-freq/governors.txt
+
+        :param cpus: The list of CPU for which the governor is to be set.
+        """
+        online_cpus = self.target.list_online_cpus()
+        for cpu in cpus:
+            if cpu not in online_cpus:
+                continue
+            self.set_governor(cpu, governor, kwargs)
```

Comment:
why not just

    for cpu in online_cpus:
        self.set_governor(cpu, governor, kwargs)
?